akvo/akvo-rsr
# [#2403] Clicking on Read more in project summary broken sometimes

When the summary ends with a list, or anything that creates a child node in
the parent summary node from the markdown, the logic of showing full summary
instead of truncated summary was broken because parentNode of the link element
wasn't the intended one.

**NOTE:** This is a PR for the 3.20 release. 

## Test Plan

- Use the summary text from [this project](http://rsr.test.akvo.org/en/project/5277/) in a test project. 
- The summary should get truncated
- Clicking the Read more button should show the full summary. 
- Verify that the Fold in button again shows the truncated summary.
